frontbackend/settings.py

```diff
@@ -0,0 +1,29 @@
+"""Default settings for the Front email backend."""
+from django.conf import settings
```

Comment:
Not quite sure why we need to duplicate the settings code? Could be useful to add in a comment if there's a design reason for it.

Reply:
Ah, gotcha. So just a default value if it's used in another project 👍